setup/mail-postfix.sh

```diff
@@ -215,6 +215,83 @@ tools/editconf.py /etc/default/postgrey \
 tools/editconf.py /etc/postfix/main.cf \
 	message_size_limit=134217728
 
+# ### MTA-STS - SMTP Mail Transfer Agent Strict Transport Security - SETUP
+# See: https://github.com/mail-in-a-box/mailinabox/pull/1556
+# create the MTA-STS policy
+mkdir -p /var/lib/mailinabox/
+cat > /var/lib/mailinabox/mta-sts.txt << EOF
+version: STSv1
+mode: enforce
+mx: \$PRIMARY_HOSTNAME
+max_age: 86400
+EOF
+chmod a+r /var/lib/mailinabox/mta-sts.txt
+
+# install the postfix MTA-STS resolver
+/usr/bin/pip3 install postfix-mta-sts-resolver
+# add a user to use solely for MTA-STS resolution
+useradd -c "Daemon for MTA-STS policy checks" mta-sts -s /sbin/nologin
```

Comment:
Looks like /usr/sbin/nologin is used in 18.04

```
root@m:~# which nologin
/usr/sbin/nologin
root@m:~# grep mta-sts /etc/passwd
mta-sts:x:1002:1002:Daemon for MTA-STS policy checks:/home/mta-sts:/sbin/nologin
```

Reply:
Fixed https://github.com/mail-in-a-box/mailinabox/pull/1556/commits/f71bd61fba1c79de5035cef6a0e69109ee19879b